cmd/otelcol/Dockerfile

```diff
@@ -45,6 +51,7 @@ FROM scratch
 ENV SPLUNK_BUNDLE_DIR=/usr/lib/splunk-otel-collector/agent-bundle
 ENV SPLUNK_COLLECTD_DIR=${SPLUNK_BUNDLE_DIR}/run/collectd
 ENV PATH=${PATH}:${SPLUNK_BUNDLE_DIR}/bin
+ENV PATH=${PATH}:${SPLUNK_BUNDLE_DIR}/jre/bin
```

Comment:
I wonder if adding `ENV JAVA_HOME=${SPLUNK_BUNDLE_DIR}/jre` helps with anything?

Reply:
I checked and it's not set by default. If everything is currently working as expected I don't know if it's necessary, but also don't think it'd hurt. Do you think there's any specific benefit of setting it?

Reply:
Looks like the it gets set for the smart agent (https://github.com/signalfx/splunk-otel-collector/blob/main/internal/signalfx-agent/pkg/core/config/config.go#L210), but wasn't sure if the jmx receiver needed it as well.  I guess if everything is working as expected without it, let's not fix things that aren't broken.